### ISSUE TITLE
Error Parsing SQLITE INSERT

### ISSUE THREAD

**Issue by jalcine** [san650/pretty_print_formatter#16]
Similar to https://github.com/san650/pretty_print_formatter/issues/9:

```sql
ERROR PARSING {1, :sql_lexer, {:illegal, ';'}} -- INSERT INTO "entries_syndication_results" ("entry_id","syndication_result_id") VALUES (?1,?2) ;--RETURNING ON INSERT "entries_syndication_results","id" [<<221, 88, 232, 72, 119, 59, 77, 141, 163, 247, 33, 23, 111, 183, 66, 46>>, <<147, 211, 213, 116, 4, 194, 65, 181, 134, 196, 32, 235, 187, 163, 106, 236>>]
```

**Comment by san650** [comment]
Fixed by #17

**Comment by san650** [comment]
Fixed in v0.1.6

**Comment by jalcine** [comment]
Epic, thanks!

(Originally published at: https://v2.jacky.wtf/post/37d228b3-1f1e-4530-ad0a-47143254aa49)
